ADRs/0003-adopt-libp2p-peer-ids.md

```diff
@@ -0,0 +1,98 @@
+# Adopting Libp2p PeerIDs <!-- omit in toc -->
+
+- Status: in review
+- Deciders: Pocket Network team
+- Date: 2023-04-14
+
+**Table of Contents**
+
+- [Summary](#summary-)
+- [Problem Statement](#problem-statement-)
+- [Background](#background-)
+- [Decision Drivers](#decision-drivers-)
+- [Considered Options](#considered-options-)
+- [Decision Outcome](#decision-outcome-)
+    - [Positive Consequences](#positive-consequences-)
+    - [Negative Consequences](#negative-consequences-)
+- [Pros and Cons of the Options](#pros-and-cons-of-the-options-)
+    - [Adopting Libp2p identity/IDs](#adopting-libp2p-identityids)
+    - [Custom identity scheme](#custom-identity-scheme)
+- [References](#references-)
+
+## Summary 
+
+Facing the need for a unique and verifiable identifier for nodes in the peer-to-peer network, we decided to adopt Libp2p identity/IDs and neglect alternative identity schemes, to achieve a reliable and secure identification system, accepting the additional dependency and integration work, because Libp2p provides a proven solution with robust security features.
+
+## Problem Statement 
```

Comment:
Do you think it's worth mentioning that this ADR builds on top of #32 to adopt multiaddr and LibP2PPeerIDs use multiaddr underneath?